i18n/es/developer_guide.md

```diff
@@ -180,105 +180,105 @@ This Specification is distributed in the hope that it will be useful, but WITHOU
-
-# Governance Process
-
-![Governance Process](https://getmonero.org/blog/assets/2015-year-in-review/governance-process.jpg)
+## Lenguaje
+
+Las palabras "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", y "OPTIONAL" en este documento, deben ser interpretadas como esta descrito en el RFC 2119.
+
+## Metas
+
+C4 está hecho para ser un modelo óptimo de colaboración para proyectos de software de código abierto. Y se especifican los siguientes objetivos:
+
+- Maximizar la escala y la diversidad de la comunidad en torno a un proyecto, reduciendo la fricción para los nuevos colaboradores y creando un modelo de participación escalado con fuertes retroalimentaciones positivas;
+- Para aliviar las dependencias de individuos clave, separando diferentes conjuntos de habilidades para que haya un mayor grupo de competencia en cualquier dominio requerido;
+- Permitir que el proyecto se desarrolle de forma más rápida y precisa, aumentando la diversidad del proceso de toma de decisiones;
+- Apoyar el ciclo de vida natural de las versiones del proyecto, desde el experimental hasta el estable, permitiendo la experimentación segura, el fracaso rápido y el aislamiento del código estable;
+- Reducir la complejidad interna de los repositorios de proyectos, facilitando así la participación de los contribuyentes y reduciendo el margen de error;
+- Reforzar la propiedad colectiva del proyecto, lo que aumenta el incentivo económico a los contribuyentes y reduce el riesgo de secuestro por entidades hostiles.
+
+## Diseño
+
+### Preliminares
+
+- El proyecto USARÁ el sistema de control de revisión distribuido git.
+- El proyecto ESTARÁ alojado en github.com o equivalente, llamado "Plataforma".
+- El proyecto USARÁ utilizar la plataforma de Tracker de issues.
+- El proyecto DEBERÁ tener directrices claramente documentadas para el estilo de código.
+- Un "Colaborador" es una persona que desea proporcionar un parche, siendo un conjunto de compromisos que resuelven algún problema claramente identificado.
+- Un "Mantenedor" es una persona que fusiona parches al proyecto. Los mantenedores no son desarrolladores; Su trabajo es hacer cumplir el proceso.
+- Los colaboradores NO TENDRÁN acceso de commit al repositorio a menos que sean también Mantenedores.
+- Los mantenedores TENDRÁN tener acceso de commit al repositorio.
+- Todos, sin distinción ni discriminación, TENDRÁN el mismo derecho de convertirse en un Colaborador en los términos de este contrato.
+
+### Licencia y propiedad
+
+- El proyecto DEBE utilizar una licencia compartida, como la GPLv3 o una variante de la misma (LGPL, AGPL), o la MPLv2.
+- Todas las contribuciones al código fuente del proyecto ("parches") DEBERÁ utilizar la misma licencia que el proyecto.

```

Comment:
"DEBERÁN" - Found [this answer](http://udep.edu.pe/castellanoactual/duda-resuelta-deberan/) to comment about this actual change.